wire/Makefile

```diff
@@ -85,6 +85,13 @@ wire/gen_onion_wire.h: wire/gen_onion_wire_csv $(WIRE_BOLT_DEPS) wire/Makefile
 wire/gen_onion_wire.c: wire/gen_onion_wire_csv $(WIRE_BOLT_DEPS) wire/Makefile
 	$(BOLT_GEN) -s --expose-tlv-type=tlv_payload --page impl ${@:.c=.h} onion_type < $< > $@
 
+# Some messages that are common among all daemons
+wire/gen_common_wire.h: wire/common_wire_csv $(WIRE_BOLT_DEPS) wire/Makefile
+	$(BOLT_GEN) -s --page header $@ common_wire_type < $< > $@
+
+wire/gen_common_wire.c: wire/common_wire_csv $(WIRE_BOLT_DEPS) wire/Makefile
+	$(BOLT_GEN) -s --page impl ${@:.c=.h} common_wire_type < $< > $@
```

Comment:
TODO: enhance devtools/decodemsg to decode internal messages.  That is not only useful, if it's constructed as a giant switch() statement, it also ensures we don't have any duplicate message numbers anywhere (otherwise it won't compile).

Meanwhile, we should probably have such a switch statement somewhere in the source (even if it's just manually maintained) to force this check?